src/webresource-handler/index.ts

```diff
@@ -151,7 +157,9 @@ export const getUploaderMiddlware = (
 			completeUploads.push(
 				(async () => {
 					try {
-						if (!(await isFileInValidPath(fieldname, req))) {
+						if (
+							!(await isFileInValidPath(fieldname, req, parsedOdataRequest))
```

Comment:
As a later improvement there's a lot of work done in `isFileInValidPath` that would be nice to avoid duplicating if there are multiple uploads in a single request. I don't know if the duplication is avoidable but I suspect so

Reply:
Added in my todo list for webresources